views/game_result.erb

```diff
@@ -0,0 +1,5 @@
+<div class="results">
+  <h2> <%= @game.player %>, you chose <%= @game.player_select %> </h2>
```

Comment:
opinion: from what I understand, our erb files should be light in code, so it would be good to call the method on @game outside of this file